templates/CRM/Admin/Form/Setting/Search.tpl

```diff
@@ -68,11 +68,28 @@
             <td>{$form.smartGroupCacheTimeout.html}<br />
                 <span class="description">{ts}The number of minutes to cache smart group contacts. We strongly recommend that this value be greater than zero, since a value of zero means no caching at all. If your contact data changes frequently, you should set this value to at least 5 minutes.{/ts}</span></td>
         </tr>
+        <tr class="crm-search-setting-form-block-default_quicksearch_option">
+            <td class="label">{$form.default_quicksearch_option.label}</td>
+            <td>
+                {$form.default_quicksearch_option.html}
+                <p class="description">{$setting_descriptions.default_quicksearch_option}</p>
+            </td>
+        </tr>
         <tr class="crm-search-setting-form-block-quicksearch_options">
             <td class="label">{$form.quicksearch_options.label}</td>
             <td>
-                {$form.quicksearch_options.html}
+                <ul class="crm-search-setting-form-block-quicksearch_options-list{if $quicksearch_options_sortable} crm-sortable{/if}">
```

Comment:
I believe coleman suggestion on the ticket was to move this to a new generic template like here : 
https://github.com/civicrm/civicrm-core/tree/master/templates/CRM/Core
Similar to BillingBlock.tpl or DatePickerRange.tpl and for you to just include the this generic template here using smarty "include" : https://www.smarty.net/docsv2/en/language.function.include.tpl .

And to ensure that the new template is generic and will work for any other form if needed, You can pass the checkbox list through a variable that you pass to the template as explained in smarty docs above, so if any one want to use it for another form they can just include the new template and pass the list of checkbox options to the template.